lib/mi_bridges/driver.rb

```diff
@@ -160,6 +160,8 @@ def level
     end
 
     def save_error(e, page)
+      return if latest_drive_attempt.nil?
```

Comment:
might also be a good time to add a `NullDriveAttempt`

Reply:
Let's explore that and see about getting that in soon. I do want to get this in before I sign off for the day. So I'll merge this in and see how things are.